R/zchunk_L102.water.supply.unlimited.R

```diff
@@ -14,7 +14,7 @@
 #' @importFrom tidyr gather spread
 #' @author YourInitials CurrentMonthName 2017
 #' @export
```

Comment:
All information in the header (descriptive comment at top, etc.) needs to be filled out.